sfdx-source/apex-common/main/classes/fflib_QueryFactory.cls

```diff
@@ -662,6 +663,14 @@ public class fflib_QueryFactory { //No explicit sharing declaration - inherit fr
 		return setOrdering(ordr);
 	}
 
+	/**
+	 * @param addAllRows whether an ALL ROWS clause will be added to the resulting query.
+	**/
+	public fflib_QueryFactory setAllRows(Boolean addAllRows){
```

Comment:
@1903Daniel -- I would suggest just a no parameter method.  Simply `fflib_QueryFactory setAllRows()` 

In this case, you would only ever flip the `allRows` class variable to `true`.  Since it defaults to `false`, there is no real benefit to calling this method with a parameter of `false`.  

I would suggest changing this method to no parameters and then simply set the `allRows` member variable to true.

Reply:
@ImJohnMDaniel Thanks for the suggestion. I thought about this as well. However, after you `deepClone()` an `fflib_QueryFactory` with the variable already set to `true`, you wouldn't be able to flip the variable back to `false` again. Do you think this is a valid use case? If not, I'll happily change `setAllRows` to a no params method.

Reply:
@1903Daniel, I cannot say that I have ever done a `deepClone()` of a QueryFactory instance.  Let's proceed with the change to a no param method.  If someone comes across this issue in the future, then we can deal with it then.  Cheers!

Reply:
@ImJohnMDaniel Done